docs/optimizations/FileLocalization.md

```diff
@@ -1,5 +1,7 @@
 # The 'localization_optional' Optimization
 
+Available in Cromwell version 33 and higher.
```

Comment:
👍

Reply:
Could you make this again as a top-level comment (or make a "approve" review)?

Looks like pullapprove doesn't count review-comment thumbs :'(